cloud/tasks/logging/journald_logger.go

```diff
@@ -88,35 +174,40 @@ func (l *journaldLogger) structuredLogWithCaller(
 	msg string,
 	fields ...log.Field,
 ) {
+	ff := newJournaldLoggerFileds(fields...)
+
+	if len(ff.messageFields) > 0 {
+		msg = serializeFields(ff.messageFields) + " => " + msg
```

Comment:
Error "=>" appears around the fields and also around the message body. Like this:
Prefix => fields => message body => error.